app/views/documents/attachment/_thumbnail_document.html.erb

```diff
@@ -0,0 +1,3 @@
+<svg class="gem-c-attachment__thumbnail-image" version="1.1" viewBox="0 0 99 140" width="99" height="140" aria-hidden="true">
```

Comment:
It might be nice to add a `<%# comment %>` to the beginning of these thumbnail files (which should be possible since they're `.erb` rather than `.svg`) just to note where the SVGs were originally sourced from.

e.g. something like:
```suggestion
<%# Source: https://github.com/alphagov/govuk_publishing_components/blob/1facbe66c93fcfc15f77cbcde00d90b20874765c/app/views/govuk_publishing_components/components/attachment/_thumbnail_document.html.erb %>
<svg class="gem-c-attachment__thumbnail-image" version="1.1" viewBox="0 0 99 140" width="99" height="140" aria-hidden="true">
```